source/user-manual/capabilities/vulnerability-detection/how-it-works.rst

```diff
@@ -18,25 +18,40 @@ We maintain the integrity of our vulnerability data by doing the following addit
+Then, the module scans the software inventory of the endpoints using the latest available vulnerabilities information. The detection process looks for vulnerable packages in the inventory databases. These inventories are unique to each agent.
 
-A package is labeled as vulnerable when its version matches those within the affected range of a CVE. Alerts show the results, and the module stores the findings in a per-agent vulnerability inventory. This inventory contains the current state of every agent and includes vulnerabilities that have been detected and not resolved. Users can :doc:`query the inventory </user-manual/capabilities/system-inventory/index>` to check for alerts and vulnerability information.
+A package is labeled as vulnerable when its version matches those within the affected range of a CVE. Alerts show the results, and the module stores the findings in a general vulnerability inventory that can be filtered per-agent. This inventory contains the current state of every agent and only includes vulnerabilities that have been detected and not resolved. Users can :ref:`query the inventory <vuln_det_poc_alert_visualize>` to check for alerts and vulnerability information.
 
-For Microsoft Windows systems and specific Microsoft products, the Vulnerability Detection module has the ``hotfixes`` option in the syscollector settings on the Wazuh agent. Using this option enables the module to detect packages that the user has patched. When the Vulnerability Detection module detects a patch, it uses the information provided by Microsoft to decide if the patch has resolved the CVEs. Then, it removes them from the list of vulnerabilities.
+For Microsoft Windows systems and specific Microsoft products, the Vulnerability Detection module has the :ref:`wodle_syscollector_hotfixes` option in the :ref:`syscollector <system_inventory>` settings on the Wazuh agent. Using this option enables the module to detect packages that the user has patched. When the Vulnerability Detection module detects a patch, it uses the information provided by Microsoft to decide if the patch has resolved the CVEs. Then, it removes them from the list of vulnerabilities.
+
+.. _vuln_det_alert_generation:
 
 Alert generation
 ----------------
 
-The Vulnerability Detection module generates alerts when it detects new vulnerabilities or when users fix identified vulnerabilities. You can see this workflow in the diagram below.
+The Vulnerability Detection module generates alerts when it detects new vulnerabilities or when users fix identified vulnerabilities.
+However, to enhance the reliability of Wazuh platform, the module doesn't always generate alerts due to a vulnerability detected or solved. An alert means that something has just happened, and that isn't always the case:
```

Comment:
"due to a vulnerability detected or solved." I didn't understood this part

Reply:
I've updated the sentence.